### ISSUE TITLE
Link to github from http://www.peervpn.net/

### ISSUE THREAD

**Issue by Thordin** [peervpn/peervpn#4]
There should be a link to this github from http://www.peervpn.net/

**Comment by jeteon** [comment]
Yeah. I was pleasantly surprised to come across the repo today. It really boosted my confidence in the project as the website is pretty spartan.

**Comment by sequoiar** [comment]
+1
